examples/autotagger/src/locations/Sidebar.spec.tsx

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import Sidebar from './Sidebar';
 import { render } from '@testing-library/react';
 import { mockCma, mockSdk } from '../../test/mocks';
+import { useAutoResizer } from '@contentful/react-apps-toolkit';
 
 vi.mock('@contentful/react-apps-toolkit', () => ({
   useSDK: () => mockSdk,
   useCMA: () => mockCma,
+  useAutoResizer: vi.fn(),
 }));
 
 describe('Sidebar component', () => {
   it('Component text exists', () => {
     const { getByText } = render(<Sidebar />);
 
-    expect(getByText('Hello Sidebar Component (AppId: test-app)')).toBeInTheDocument();
```

Comment:
It's weird that this test wasn't failing before (or maybe these tests just aren't a part of the CI checks for the examples?) but the component doesn't actually render this test, looks like this test was just copy/pasted from other tests for their Sidebar, but this test wasn't actually a reflection of the Sidebar content for this particular example. I updated it it to find something that the component actually renders.